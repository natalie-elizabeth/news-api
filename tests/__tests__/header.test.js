import { shallow, render, mount } from 'enzyme';
import React from 'react';
import Header from '../../src/app/components/header';

describe('Header', () => {
  it('should display an image', () => {
    const wrapper = shallow(<img />);
    expect(wrapper.find('img').node).toBeDefined();
  });
  it('should render an image', () => {
    const wrapper_ = shallow(<img src="./src/images/title.jpg" />);
    expect(wrapper_).toMatchSnapshot();
  });
  it('should display homepage image', () => {
    const wrapper = shallow(<img />);
    expect(wrapper.containsMatchingElement(<img src='./src/images/title.jpg' style={{ width: '1140px', height: '200px' }} />));
  });
  it('should render image in the divs', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.containsMatchingElement(
      <div className="headers">
        <img className='main-logo' src="./src/images/titlelogo.jpg" />
      </div>
    ));
  });

});




