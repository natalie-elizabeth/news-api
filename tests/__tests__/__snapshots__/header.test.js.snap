// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render an image 1`] = `
<img
  src="./src/images/title.jpg"
/>
`;
